keystore/downloadkeys.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#Script used for download aosp and nxp platform keys.
+
+BASE_PWD=$PWD
+
+#Create & setup aosp directory
+
+mkdir aosp && cd aosp
+curl -O https://raw.githubusercontent.com/aosp-mirror/platform_build/android-11.0.0_r48/target/product/security/platform.x509.pem
+curl -O https://raw.githubusercontent.com/aosp-mirror/platform_build/android-11.0.0_r48/target/product/security/platform.pk8
+
+cd $BASE_PWD
+
+#Create & setup nxp directory
+
+mkdir nxp && cd nxp
+curl -O https://source.codeaurora.org/external/imx/android-imx/device/fsl/plain/common/security/platform.x509.pem
+curl -O https://source.codeaurora.org/external/imx/android-imx/device/fsl/plain/common/security/platform.pk8
```

Comment:
```suggestion
curl -O https://source.codeaurora.org/external/imx/android-imx/device/fsl/plain/common/security/platform.x509.pem?h=android-11.0.0_2.6.0
curl -O https://source.codeaurora.org/external/imx/android-imx/device/fsl/plain/common/security/platform.pk8?h=android-11.0.0_2.6.0
```

It's preferable to use tags, as master can change (e.g. keys may be moved to a different directory)